lib/amazon_ssa_support/miq_ec2_vm/miq_ec2_ebs_image.rb

```diff
@@ -2,7 +2,7 @@
 
 module AmazonSsaSupport
   class MiqEC2EbsImage < MiqEC2EbsVmBase
-    def initialize(ec2_obj, host_instance, ec2)
+    def initialize(ec2_obj, host_instance, ec2, ost = nil)
```

Comment:
From the usage, it seems `ost` only contains `scanData`, which isn't used by `MiqVm`.
Given it's also not used by the various `MiqEC2*` classes, why pass it in at all?

Same question applies to all of the following.

Reply:
Wrong. MiqVm uses ost parameter, that's the fix of this PR.

Reply:
https://github.com/hsong-rh/amazon_ssa_support/blob/3a6e25fc432d73dc56877a193b5bc62e7675feab/lib/amazon_ssa_support/miq_ec2_vm/miq_ec2_ebs_vmbase.rb#L61

This is the place MiqVm uses it. The scan items are set here.